### ISSUE TITLE
Detect extension owner change

### ISSUE THREAD

**Issue by gitbees165** [Rayquaza01/changelogger#37]
Nice extension by the way.
Would it be possible to detect change of owner, aside from the usual changelogs? Found a chrome extension which does a great function. https://github.com/classvsoftware/under-new-management?tab=readme-ov-file

**Comment by Rayquaza01** [comment]
Yes, this should be possible. The [addons details API](https://mozilla.github.io/addons-server/topics/api/v4_frozen/addons.html#get--api-v4-addons-addon-(int-id|string-slug|string-guid)-) has an authors field, which returns an array of the extension's authors. We can just save a copy of the authors array and check if it's changed when an update happens.

Example of the authors array from https://addons.mozilla.org/api/v4/addons/addon/changelogger@r01/
![Screenshot from 2024-03-09 15-18-00](https://github.com/Rayquaza01/changelogger/assets/22138871/f950724b-4a36-4b41-9a39-38e147c6ad5b)

**Comment by gitbees165** [comment]
Knowing whether the owner changed is good from an online security perspective, for the changelogger extension. +1 for safety
